.ci/CI/src/modules/Bootloader/JobConfig.jl

```diff
@@ -60,6 +60,55 @@ function get_unit_test_configs(args::Dict{String, Any})::Vector{Tuple{JuliaVersi
     return unit_test_types
 end
 
+"""
+Returns all integration tests configurations configured by script arguments and environment variables.
+"""
+function get_integration_test_configs(args::Dict{String, Any})::Vector{Tuple{JuliaVersionType, TestPlatform}}
+    integ_test_types = Vector{Tuple{JuliaVersionType, TestPlatform}}()
+
+    if !is_integ_tests(args)
+        return integ_test_types
+    end
+
+    for julia_version in get_integration_test_julia_versions()
+        if julia_version == "nightly" || julia_version == "rc"
+            continue
+        else
```

Comment:
Since this won't return from the if clause anyway, please make it an `if ... end` to save one indentation of the rest of the block.